### DISCUSSION TITLE
Code not working

### DISCUSSION THREAD

**Discussion by joe44824** [osamaqarem/react-native-image-colors#55] [Q&A] [answered]
Hi sir. I am not able to produce the result. Please help to take a look.

`  const [color, setColor] = useState(null);

  useEffect(() => {
    async function fetchColor() {
      const result = await ImageColors.getColors(uri, {
        fallback: '#228B22',
        cache: true,
        key: 'unique_key',
      });
      switch (result.platform) {
        case 'android':
          // android result properties
          const vibrantColor = result.vibrant;
          setColor(vibrantColor);
          break;
        case 'web':
          // web result properties
          const lightVibrantColor = result.lightVibrant;
          setColor(lightVibrantColor);
          break;
        case 'ios':
          // iOS result properties
          const primaryColor = result.primary;
          setColor(primaryColor);
          break;
        default:
          throw new Error('Unexpected platform key');
      }
    }
    fetchColor();
  }, []);`

**Comment by osamaqarem** [comment, marked as answer]
Hey, it's not obvious what the problem could be from the info given. 
In this case please go the the issues tab and press 'new issue' -> choose the bug template -> fill up the title and info -> submit 
I could then take a closer look.
